pathfinder/tests/fixtures/accounts.py

```diff
@@ -22,24 +23,24 @@ def faucet_address(faucet_private_key: str):
 
 
 @pytest.fixture(scope='session')
-def private_keys() -> List[str]:
+def private_key_seeds() -> List[str]:
```

Comment:
Why did you rename this? It's still a list of actual Ethereum private keys (hex-encoded). I get your critique that this isn't the ECDSA private key but still, we call this the private key in Ethereum. See the yellow paper if you don't believe me.

Reply:
No offence, it's just a list of hashes of an ascending sequence of natural numbers, encoded as hex. Calling it private keys seems dangerous. I think even calling the result of privatekey.from_hex() on it a private key is dangerous. At some point someone might start using this, we should make it hard to make mistakes.

Reply:
They are private keys because of what they are used for, not because of how they were created. We're using deterministically generated private keys. Nothing wrong with that in a test setup.

There are secure and insecure private keys. These are definitely the latter but they are still private keys.

Reply:
This still holds after the recent push. Please rename to `private_key` because that's what it is and how it is used.

Reply:
Ok, done.